docs-ref-mapping/reference-preview.yml

```diff
@@ -317,6 +317,11 @@
     href: ~/docs-ref-services/preview/ai-textanalytics-readme.md
     children:
     - azure-ai-textanalytics
+  - name: Translator
+    uid: azure.python.sdk.landingPage.services.Translator.pre
+    href: ~/docs-ref-services/preview/ai-translation-document-readme.md
```

Comment:
This looks great @kristapratico . The change is g2g. We just need to mirror this with {moniker} (and no `.pre` suffix) to reference-unified.